api/capi/include/ml-api-common.h

```diff
@@ -47,6 +51,35 @@ typedef enum {
 /**
  * @}
  */
+
+#else
+
+#include <errno.h>
+
+/**
+ @ref: https://gitlab.freedesktop.org/dude/gst-plugins-base/commit/89095e7f91cfbfe625ec2522da49053f1f98baf8
+ */
+#if !defined(ESTRPIPE)
+#define ESTRPIPE EPIPE
+#endif /* !defined(ESTRPIPE) */
+
+/**
+ * @brief Enumeration for the error codes of NNStreamer for None-tizen platform
+ */
+typedef enum {
+  ML_ERROR_NONE                 = 0, /**< Success! */
```

Comment:
let's define TIZEN_ERROR_* than ML_ERROR_* again
as adding a new error code, in this case, would require adding it twice in both cases.

Reply:
This variant is reflection of 
https://github.com/nnstreamer/nntrainer/pull/243#discussion_r447501431

As `Tizen_*` is platform dependent, if we are open to bind other platform dependent error code, IMO, it make sense to keep it this way.

Reply:
cool.